### DISCUSSION TITLE
[9주차] 온라인 스터디

### DISCUSSION THREAD

**Discussion by ChaerinYu** [no-study-no-future/interview-study#65] [General]
~ 20:15 각자 PR리뷰
~ 21:00 발표
~ 21:10 회고 (좋았던 점, 건의사항, 느낀점 등)

**Comment by ChaerinYu** [comment]
### 발표
1. 유채린
 * [ETC/DesignPattern] Proxy Pattern 프록시 패턴 #62 
2. 이주형
 * [Spring] Spring과 Spring Boot, 빌드 관리 도구 #64 
3. 이동호
 * [Network] CORS #66

**Comment by ChaerinYu** [comment]
### 회고 (a.k.a. 출석체크)
* 참석 확인용이므로 `감사합니다!` 만 작성해주셔도 좋아요 😊
* 좋았던 점
* 건의사항
* 느낀점

**Reply by ChaerinYu** [reply]
오늘도 고생많으셨어요~! 🙂

**Reply by leeejuhyeong** [reply]
담주는 빡센거..

**Reply by ChaerinYu** [reply]
@Kodo92 ㅊㅊ
